### PR TITLE
adding resize ability 

### PR DESCRIPTION
**Pull request by tcarlsen**
fixes issue #2

### CODE REVIEW

**Comment by tcarlsen** [comment]
please review @joefitzgerald

File: .jslintrc [hunk 1]
```diff
@@ -2,6 +2,7 @@
   "indent": 2,
   "node": true,
   "predef": {
-		"atom": true
+		"atom": true,
```
Comment (joefitzgerald) [inline .jslintrc]: Bad indent?

File: .jslintrc [hunk 2]
```diff
@@ -2,6 +2,7 @@
...
+    "document": true
```
Comment (joefitzgerald) [inline .jslintrc]: Bad indent?

**Comment by joefitzgerald** [comment]
LGTM. I particularly like that max height can now be something that is relative to the viewport instead of a pixel-based measurement. :+1:

File: .jslintrc (hunk 1 above)
Comment (tcarlsen) [inline .jslintrc]: Nope just adding document as a global. Using it now

File: .jslintrc (hunk 2 above)
Comment (tcarlsen) [inline .jslintrc]: Same ^^

Comment (joefitzgerald) [inline .jslintrc]: There's definitely something screwy in the indenting:

```javascript
{
  "indent": 2,
  "node": true,
  "predef": {
		"atom": true,
    "document": true
	}
}
```

**Pull request merged**